fair_mango/metrics/base.py

```diff
@@ -216,6 +222,33 @@ def __call__(self): ...
 def calculate_disparity(
     result_per_groups: np.ndarray, method: Literal["difference", "ratio"]
 ) -> dict[tuple, np.ndarray[float]]:
+    """Calculate the disparity in the scores between every possible pair in
+    the provided groups using two available methods:
+    - difference (Example: for three groups a, b, c:
+    [score_a - score_b], [score_a - score_c], [score_b - score_c])
+    - ratio (Example: for three groups a, b, c:
+    [score_a / score_b], [score_a / score_c], [score_b / score_c])
+
+    Parameters
+    ----------
+    result_per_groups : np.ndarray
```

Comment:
My bad, it is indeed a `list[dict]` after investigation.

Reply:
it seems like it has not been updated

Reply:
It is updated in another branch because it was not related to the documentation only.

PS: Many comments that are outside the scope of this PR are resolved on another branch.